packages/showcases/css/stories/components/radio-button/examples/disabled.html

```diff
@@ -0,0 +1,4 @@
+<div class="block">
+  <input type="radio" class="vtmn-radio-button" id="my-radio-button" disabled />
+  <label class="vtmn-radio-button_label" for="my-radio-button">Label</label>
```

Comment:
I think you can avoid to use a specific class here and use directly the "label" tag to make your styles.
Same for the input.